Config.h

```diff
@@ -18,14 +18,18 @@
 
 //      Parameter Name              Value   Default  Notes                                                                      Hint
 #define PINMAP                        OFF //    OFF, Choose from: MiniPCB, MiniPCB2, MaxPCB, MaxPCB2, STM32Blue,             <-Req'd
-                                          //         Ramps14, MaxESP2, MaxESP3.  Check Constants.h for more info.
+                                          //         Ramps14, MaxESP2, MaxESP3, InsteinESP1.  Check Constants.h for more info.
 
 // SERIAL PORT COMMAND CHANNELS ------------------------------------ see https://onstep.groups.io/g/main/wiki/6-Configuration#SERIAL
 #define SERIAL_A_BAUD_DEFAULT        9600 //   9600, n. Where n=9600,19200,57600,115200 (common baud rates.)                  Infreq
 #define SERIAL_B_BAUD_DEFAULT        9600 //   9600, n. See (src/HAL/) for your MCU Serial port # etc.                        Option
 #define SERIAL_B_ESP_FLASHING         OFF //    OFF, ON Upload ESP8266 WiFi firmware through SERIAL_B with :ESPFLASH# cmd.    Option
 #define SERIAL_C_BAUD_DEFAULT         OFF //    OFF, n, ON for ESP32 Bluetooth.                                               Option
 #define SERIAL_C_BLUETOOTH_NAME  "OnStep" // "On..", Bluetooth device name for ESP32.                                         Option
+#define SERIAL_D_BAUD_DEFAULT        9600 // ###req'd for Instein### Instein Drive AUX COM Port baudrate, lowest baudrates = larguer 
```

Comment:
I'd like this removed, it's just "noise" for 99% of users.
1. I try to keep all platform specific stuff out of the Config.h file.
2. Nearly all platforms are limited to 3 or less serial channels.  If a specific platform has more make it happen in the HAL.  We can document all in the Wiki.